docs/conf.py

```diff
@@ -26,12 +28,12 @@
 ]
 
 # Only build the videos on production, to speed up dev
-import os
 on_rtd = os.environ.get('READTHEDOCS') == 'True'
 on_netlify = os.environ.get('BUILD_VIDEOS') == 'True'
 on_travis = os.environ.get('TRAVIS') == 'True'
 if not on_rtd and not on_netlify and not on_travis:
     exclude_patterns.append('videos')
+REWRITE_FEED = False
```

Comment:
Setting to `True` locally I get

```
File "/home/samuel/Nosync/Repositories/WTD/www/docs/_ext/atom_absolute.py", line 37, in rewrite_atom_feed
    namespace = root.nsmap.values()[0]
TypeError: 'dict_values' object does not support indexing
```

Reply:
Hrm, probably a py3 issue, will poke at it.